src/styles/codeStyles/types.ts

```diff
@@ -0,0 +1,7 @@
+import {TextStyle, ViewStyle} from 'react-native';
+
+type CodeWordWrapperStyle = Partial<Pick<ViewStyle, 'height' | 'justifyContent'>>;
+type CodeWordStyle = Partial<Pick<TextStyle, 'height' | 'top'>>;
+type CodeTextStyle = Partial<Pick<TextStyle, 'lineHeight'>>;
```

Comment:
```suggestion
type CodeWordWrapperStyles = ViewStyle;
type CodeWordStyles = TextStyle;
type CodeTextStyles = TextStyle;
```

I think we can relax a little bit on these styles @BartoszGrajdek , since in their context they are related to general styles that could be applied to the code component.

Also, could we rename to `...Styles` at the end to keep consistency?

Reply:
@fabioh8010 should be ok now :)